README.md

```diff
@@ -59,7 +59,6 @@ To get a local copy up and running follow these simple example steps.
 
```

Comment:
### Your File Structure
![should](https://user-images.githubusercontent.com/93039359/191125228-54329f1f-eb53-4d2b-8efa-87a340e55833.png)

### Expected File Strucutre
![Screenshot from 2022-09-20 02-46-03](https://user-images.githubusercontent.com/93039359/191125495-7a10021c-182e-408e-884d-42815a23b29f.png)

- [ ] Please make your file structure as shown in the expected file structure to organize your project so in future you won't face any problems while expanding your whole project :blush Kindly make your redux folder outside the components folder :+1:

Reply:
Okay, thanks for the detailed review.

Reply:
Directories Arrangements fixed